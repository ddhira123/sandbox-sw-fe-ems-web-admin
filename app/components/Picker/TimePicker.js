import { TimePicker as TmPick } from 'antd';
import styled from 'styled-components';

const TimePicker = styled(TmPick)`
  border-radius: 10px !important;
  height: 40px !important;
`;

export default TimePicker;
