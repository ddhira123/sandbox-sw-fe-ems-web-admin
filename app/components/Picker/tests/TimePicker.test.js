import React from 'react';
import { render } from '@testing-library/react';
import TimePicker from '../TimePicker';

describe('TimePicker', () => {
  it('should render correctly', () => {
    const app = render(<TimePicker />);
    expect(app).toMatchSnapshot();
  });
});
