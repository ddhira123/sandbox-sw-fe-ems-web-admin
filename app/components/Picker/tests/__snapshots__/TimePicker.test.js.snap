// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimePicker should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ant-picker TimePicker-sc-1suwojg-0 cPgovW"
      >
        <div
          class="ant-picker-input"
        >
          <input
            autocomplete="off"
            placeholder="Select time"
            readonly=""
            size="10"
            title=""
            value=""
          />
          <span
            class="ant-picker-suffix"
          >
            <span
              aria-label="clock-circle"
              class="anticon anticon-clock-circle"
              role="img"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="clock-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M686.7 638.6L544.1 535.5V288c0-4.4-3.6-8-8-8H488c-4.4 0-8 3.6-8 8v275.4c0 2.6 1.2 5 3.3 6.5l165.4 120.6c3.6 2.6 8.6 1.8 11.2-1.7l28.6-39c2.6-3.7 1.8-8.7-1.8-11.2z"
                />
              </svg>
            </span>
          </span>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ant-picker TimePicker-sc-1suwojg-0 cPgovW"
    >
      <div
        class="ant-picker-input"
      >
        <input
          autocomplete="off"
          placeholder="Select time"
          readonly=""
          size="10"
          title=""
          value=""
        />
        <span
          class="ant-picker-suffix"
        >
          <span
            aria-label="clock-circle"
            class="anticon anticon-clock-circle"
            role="img"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="clock-circle"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
              />
              <path
                d="M686.7 638.6L544.1 535.5V288c0-4.4-3.6-8-8-8H488c-4.4 0-8 3.6-8 8v275.4c0 2.6 1.2 5 3.3 6.5l165.4 120.6c3.6 2.6 8.6 1.8 11.2-1.7l28.6-39c2.6-3.7 1.8-8.7-1.8-11.2z"
              />
            </svg>
          </span>
        </span>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
